docs/book/src/reference/versions.md

```diff
@@ -150,16 +153,20 @@ Each Cluster API contract version defines a set of rules a provider is expected
 Those rules can be in the form of CustomResourceDefinition (CRD) fields and/or expected behaviors to be implemented.
 See [provider contracts](../developer/providers/contracts/overview.md)
 
-Each Cluster API release supports only one contract version, and by convention the supported contract version matches
+Each Cluster API release supports one contract version, and by convention the supported contract version matches
 the newest API version in the same Cluster API release.
 
-| Contract Version | Status    | Supported Until                                                               |
-|------------------|-----------|-------------------------------------------------------------------------------|
-| v1beta1          | Supported | After a newer API contract will be released                                   |
-| v1alpha4         | EOL       | EOL since 2023-12-05 - v1.6.0 release date; removal planned for v1.13, Apr 26 |
-| v1alpha3         | EOL       | EOL since 2023-07-25 - v1.5.0 release date; removal planned for v1.13, Apr 26 |
+A contract version might be compatible with older contract versions; compatibility for older contract versions will 
+be dropped when the older contract version will be considered EOL.
 
-See [11919](https://github.com/kubernetes-sigs/cluster-api/issues/11919) for details about the v1alpha3/v1alpha4 removal plan.
+| Contract Version | Compatible with contract versions | Status      | Supported Until                                            |
+|------------------|-----------------------------------|-------------|------------------------------------------------------------|
+| v1beta2          | v1beta1                           | Supported   | After a newer API contract will be released                |
+| v1beta1          |                                   | Deprecated  | Deprecated since CAPI v1.11, EOL planned for v1.14, Aug 26 |
+| v1alpha4         |                                   | EOL         | EOL since 2023-12-05 - v1.6.0 release date                 |
```

Comment:
Why are we dropping the removal info from this table?

Reply:
Not exactly sure what do you mean, but old contracts are already removed; v1beta1 is now deprecated and there is a link to the removal plan right below the table

Reply:
This PR changed this table from
```
| v1alpha4         | EOL       | EOL since 2023-12-05 - v1.6.0 release date; removal planned for v1.13, Apr 26 |
| v1alpha3         | EOL       | EOL since 2023-07-25 - v1.5.0 release date; removal planned for v1.13, Apr 26 |
```

to

```
| v1alpha4 | | EOL | EOL since 2023-12-05 - v1.6.0 release date |
| v1alpha3 | | EOL | EOL since 2023-07-25 - v1.5.0 release date |
```

i.e. we removed "removal planned for v1.13, Apr 26 "

But I think I got it now. This removal was never about the contract version, only about the apiVersion. So it just didn't make sense in this table?

Reply:
That's correct, removal was never about the contract version, it is only about the apiVersion (there is only one implementation of the contract in each CAPI version)